src/amo/routes.js

```diff
@@ -32,7 +32,15 @@ export default (
     <IndexRoute component={Home} />
     <Route path="addon/:slug/" component={DetailPage} />
     <Route path="addon/:addonSlug/reviews/" component={AddonReviewList} />
-    <Route path=":visibleAddonType/categories/" component={CategoryList} />
+    {/* These user routes are to make the proxy serve each URL from */}
```

Comment:
Indeed. I messed up resolving a conflict and should have kept these removed. Will fix!